steps/deploygate--upload-app-bitrise-step/1.0.1/step.yml

```diff
@@ -0,0 +1,110 @@
+title: DeployGate Upload
+summary: |
+  Distribute your in-development iOS/Android apps to your developers or testers
+description: |
+  This is a step to upload your iOS/Android application to DeployGate. DeployGate enable to share in-development apps with others instantly. For more details, please read references in [DeployGate.com](https://deploygate.com/?locale=en)
+website: https://github.com/DeployGate/upload-app-bitrise-step
+source_code_url: https://github.com/DeployGate/upload-app-bitrise-step
+support_url: https://github.com/DeployGate/upload-app-bitrise-step/issues
+published_at: 2018-09-26T17:29:05.7800137+09:00
+source:
+  git: https://github.com/DeployGate/upload-app-bitrise-step.git
+  commit: b08f5f49e83842ba33425d5e6237a5bdc1b52c07
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: curl
+  apt_get:
+  - name: git
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- api_key: ""
```

Comment:
you can mark sensitive inputs as `is_sensitive`, this way our website will not allow to set direct value to this input, just secret envs are allowed, which are redacted in the build logs by the bitrise-cli

Reply:
Thanks!
I will use it.